server/boot/commit.js

```diff
@@ -217,11 +220,14 @@ export default function commit(app) {
       })
       .subscribe(
         pledge => {
-          let msg = 'You have successfully stopped your pledge.';
+          let msg = `You have successfully stopped your pledge. Please remember 
+                      to cancel your recurring donation directly with 
```

Comment:
~~Please remember to cancel your recurring donation directly with the nonprofit directly if you haven't already done so.~~ Please remember to cancel your recurring donation directly with the nonprofit if you haven't already done so.

> Duplicate use of the word **directly**